docs/quickstart.rst

```diff
@@ -246,14 +246,14 @@ Delete an application
     response = client.delete_application(uuid)
 
 Docs:
-`https://docs.nexmo.com/tools/application-api/api-reference#delete <https://docs.nexmo.com/tools/application-api/api-reference#delete?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/tools/application-api/api-reference#delete <https://docs.vonage.com/tools/application-api/api-reference#delete?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.